tests/JustSaying.TestingFramework/JustSaying.TestingFramework.csproj

```diff
@@ -11,7 +11,7 @@
     <PackageReference Include="MartinCostello.Logging.XUnit" Version="0.1.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.8" />
     <PackageReference Include="NSubstitute" Version="4.2.2" />
-    <PackageReference Include="Shouldly" Version="4.0.0" />
+    <PackageReference Include="Shouldly" Version="3.0.2" />
```

Comment:
I downgraded this because v4 uses a new approval test engine called DiffEngine that triggers a type load exception for some reason, in a dependent lib called EmptyFiles (https://www.nuget.org/packages/EmptyFiles/). I couldn't get to the bottom of it so downgraded for now.

Reply:
Ah, this looks like an issue with `net5.0` and Shouldly 4.0 - it works in `netcoreapp3.1`. Will keep this downgraded till there's a fix.

Reply:
I'm trying to recreate this so I can file a bug report, but haven't been able to yet, is there a commit in this PR I should use?

Reply:
Hey 🙂, I managed to reproduce it before over here: https://github.com/gkinsman/ShouldlyRepro. It fails locally with `dotnet test` and in Rider, but succeeds when run in github actions - weird!

I was going to log a bug but hadn't quite narrowed down if it was my environment yet. Does that project build locally for you?

Reply:
Thanks @gkinsman, that was really helpful: https://github.com/shouldly/shouldly/issues/717

Reply:
We can bump this back to `4.0.1` now